run-tests.php

```diff
@@ -1665,6 +1685,9 @@ function kill_children(array $children)
     }
 }
 
+/**
+ * @return true|void
+ */
```

Comment:
Where does the `true` come from? Should be just void.

Reply:
From line 1712, but will change that to void then.